Examples/Python/tests/test_covfie.py

```diff
@@ -0,0 +1,51 @@
+import pathlib, acts, acts.examples
+from acts import covfie_conversion as cc
+
+
+def test_constant_field_conversion():
+    v = acts.Vector3(1, 2, 3)
+    af = acts.ConstantBField(v)
+    cf = cc.covfieField(af)
+    view = cc.newView(cf)
+    for x, y, z in [(0, 0, 1), (1, 1, 1), (1, 0, 2)]:
+        assert view.at(x, y, z) == [1, 2, 3]
+
+
+def test_root_field_conversion():
```

Comment:
Can this be a C++-level unit test, or is the input from ODD required for testing the functionality? I imagine this would remove the need for the majority of the python binding code as well.

EDIT: Ok I see now there are C++ unit tests. Should we go for python tests in addition though?